### PR TITLE
Remove permission dialog

### PR DESCRIPTION
**Pull request by abelgardep**
Removing the dialog to ask for read/write storage permission. With scoped storage, we do not need it anymore. We store the files in scoped and we do not need permissions to do it.

_____

### CODE REVIEW

File: owncloudApp/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -476,8 +476,7 @@
   <string name="manage_space_description">سيتم حذف الإعدادات وقواعد البيانات وشهادات الخادم من بيانات %1$s\'s نهائيًا. \n\nسيتم الاحتفاظ بالملفات التي تم تنزيلها دون تغيير.\n\nيمكن أن تستغرق هذه العملية بعض الوقت.</string>
   <string name="manage_space_clear_data">مسح البيانات</string>
   <string name="manage_space_error">تعذّر حذف بعض الملفات.</string>
-  <string name="permission_storage_access">الأذونات الإضافية مطلوبة لتحميل الملفات وتنزيلها.</string>
-  <string name="local_file_not_found_toast">لم يتم العثور على الملف في نظام الملفات المحلية</string>
+    <string name="local_file_not_found_toast">لم يتم العثور على الملف في نظام الملفات المحلية</string>
```
Comment (JuancaG05) [inline owncloudApp/src/main/res/values-ar/strings.xml:479]: It seems that the resource manager in Android Studio adds an indentation when you remove a string resource. This has happened in every file 😅 . Try to reformat the code selecting them all, and if you are not able to, I will approve it 👍

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
A tiny comment here @abelgardep, good job!

**Review by JuancaG05 — APPROVED** [review]
Good job! @abelgardep

**Comment by jesmrec** [comment]
I'd add a `Calens` entry, even being not a feature. Just for our internal tracking, since this is an old behaviour change.

**Comment by jesmrec** [comment]
Approved on my side:

- [x] Dialog removed
- [x] Sync after connecting
- [x] Transfers correctly done 

Tested with Nexus 6P v7, Pixel 2 v11

**Pull request merged**